open-budget/app/scripts/entity_details.coffee

```diff
@@ -1,14 +1,17 @@
-define(["jquery", "backbone", "models", "templates"], ($, Backbone, models, JST) ->
+define(["jquery", "underscore", "backbone", "models", "templates"], ($, _, Backbone, models, JST) ->
     class EntityDetailsView extends Backbone.View
             events:
               'click .exemption-expander': 'toggleExemptionDetails'
 
             initialize: ->
-                @model.selectedEntity.on 'change:selected', =>
```

Comment:
Here's an example:
http://www.obudget.org/g2/en/511380693#entity/511380693/2014/main

I put the comment on the change that probably broke it, but I didn't test it to make sure.